py3status/modules/clementine.py

```diff
@@ -4,6 +4,10 @@
 
 Configuration parameters:
     cache_timeout: how often we refresh this module in seconds (default 5)
+    format: string to print (default '{current}')
```

Comment:
I'd default this too `'♫ {current}'` and worry about the song format another day